MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+include plotter/templates/*
+include plotter/static/*
```

Comment:
I needed these lines in order to install the templates and static files using `python setup.py install`